docs/APIs/indexMPS.md

```diff
@@ -1,64 +1,46 @@
-   | **[ConnectedDevices](./MPSmethods/connecteddevices.md)** | Lists all devices currently connected to MPS |
-   | **[Disconnect](./MPSmethods/disconnect.md)** | Disconnects the CIRA connection for a specified guid |
-   | **[MEScript](./MPSmethods/mescript.md)** | Downloads the cira_setup.mescript from MPS |
-   | **[RootCertificate](./MPSmethods/rootcertificate.md)** | Download the MPS Root Certificate |
+   | **AllDevices** | Lists all devices known to MPS, regardless of connected status |
+   | **ConnectedDevices** | Lists all devices currently connected to MPS |
+   | **Disconnect** | Disconnects the CIRA connection for a specified guid |
+   | **MEScript** | Downloads the cira_setup.mescript from MPS |
+   | **RootCertificate** | Download the MPS Root Certificate |
 
 ### AMT Methods
 
 !!! note
     These methods are 1:1 device-specific.
 
+AMT methods allow you to send device-specific calls for things such as power actions or audit logs. Utilize these methods by setting the path to **amt** in your API call.
+
    | Method       |  Description/Usage |
    | :----------- | :------------------------ |   
-   | **[AuditLog](./MPSmethods/auditlog.md)** | Returns a requested amount or time range of Intel® AMT Audit Log data for a specified guid |
-   | **[EventLog](./MPSmethods/eventlog.md)** | Return sensor and hardware event data |
-   | **[GeneralSettings](./MPSmethods/generalsettings.md)** | View general network settings |
-   | **[GetAMTFeatures](./MPSmethods/getamtfeatures.md)** | View what AMT out-of-band features are enabled/disabled |
-   | **[HardwareInfo](./MPSmethods/hardwareinfo.md)** | Retrieve hardware information such as processor or storage  |
-   | **[PowerAction](./MPSmethods/poweraction.md)** | Perform an OOB power action |
-   | **[PowerCapabilities](./MPSmethods/powercapabilities.md)** | View what OOB power actions are available for that device |
-   | **[PowerState](./MPSmethods/powerstate.md)** | Retrieve current state of AMT device, returns a number that maps to the [PowerActions](./MPSmethods/poweraction.md) table |
-   | **[SetAMTFeatures](./MPSmethods/setamtfeatures.md)** | Enable/Disable AMT features such as KVM, SOL, and IDE-R |
-   | **[Version](./MPSmethods/version.md)** | Retrieve AMT version of device |
- 
\ No newline at end of file
+   | **AuditLog** | Returns a requested amount or time range of Intel® AMT Audit Log data for a specified guid |
+   | **EventLog** | Return sensor and hardware event data |
+   | **GeneralSettings** | View general network settings |
+   | **GetAMTFeatures** | View what AMT out-of-band features are enabled/disabled |
+   | **HardwareInfo** | Retrieve hardware information such as processor or storage  |
+   | **PowerAction** | Perform an OOB power action |
+   | **PowerCapabilities** | View what OOB power actions are available for that device |
+   | **PowerState** | Retrieve current state of AMT device, returns a number that maps to the PowerActions .md) table |
```

Comment:
Leftover text from hyperlink removal: 'returns a number that maps to the PowerActions .md)'.  

Are PowerAction codes something that can be captured and explained in Swagger? Or do we need a quick doc hyperlinked and/or just listed below on this page.

Reply:
the response is provided as an example in swagger which has those values, but I'll put it in a reference doc for easy access.